index.js

```diff
@@ -124,7 +125,7 @@ function getCoveragePercentage({ jobId, startTime, endTime }) {
     const from = encodeURIComponent(parsedStartTime);
     const to = encodeURIComponent(parsedEndTime);
     // eslint-disable-next-line max-len
-    const coverageUrl = `${sonarHost}/api/measures/search_history?component=${componentId}&metrics=coverage&from=${from}&to=${to}&ps=1`;
+    const coverageUrl = `${sonarHost}/api/measures/search_history?component=${componentId}&metrics=test_success_density,coverage&from=${from}&to=${to}&ps=1`;
```

Comment:
can we get count as well ? like failed / total ?

Reply:
yeah, update it already